modules/plugins/utility/motion/precognition/config.nix

```
@@ -0,0 +1,21 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+
+  cfg = config.vim.utility.motion.precognition;
+in {
+  config =
+    mkIf cfg.enable
+    {
+      vim.startPlugins = [
+        "precognition-nvim"
+      ];
+
+      vim.luaConfigRC.precognition = lib.nvim.dag.entryAnywhere ''
+        require('precognition').setup(${lib.nvim.lua.toLuaObject cfg.setupOpts})
+      '';
+    };
+}
```

```suggestion
  config = mkIf cfg.enable {
  	vim = {
  		startPlugins = ["precognition-nvim"];

      luaConfigRC.precognition = lib.nvim.dag.entryAnywhere ''
        require('precognition').setup(${lib.nvim.lua.toLuaObject cfg.setupOpts})
      '';
    };
  };
}
```

For minimal diffs, and a happier statix lint.

---

Resolved in 706c6ab60c457353eacb3569087f686e03cd1b26 and c20bfdd6dcf1a9f71e1e363ea96dbae0ed854574.